resources/views/event/review/create.blade.php

```diff
@@ -1,9 +1,32 @@
 @extends('layouts.app')
 
-@section('content')
-    <h1> create organization review on {{ $event->name }}</h1>
-    {!! Form::open(['url' => 'event/'.$event->id.'/review']) !!}
-        @include('event.review.partials.form', ['submitButtonText' => 'Add Review!'])
-    {!! Form::close() !!}
-    
+        @section('content')
```

Comment:
"??" ? Give me comment .
 when I pulled I found overwriting this form. What should i do.